server/service/apple_mdm.go

```diff
@@ -3602,6 +3602,20 @@ func (svc *MDMAppleCheckinAndCommandService) GetToken(_ *mdm.Request, _ *mdm.Get
 	return nil, nil
 }
 
+func (svc *MDMAppleCheckinAndCommandService) runCommandHandlers(ctx context.Context, cmdName string, result fleet.MDMCommandResults) error {
+	handlers, ok := svc.commandHandlers[cmdName]
+	if ok {
+		for _, f := range handlers {
+			if err := f(ctx, result); err != nil {
+				// TODO: should we run as many as we can? if so we have to collect into a multierror
```

Comment:
I can address this in a subsequent PR, since this is the existing behavior

Reply:
Gonna merge on top of this and @lucasmrod can stack on top.

Reply:
> Guessing we'll need this for automatic VPP updates anyway? 

Do we? Will there be any commands that will be processed by more than one handler?

Reply:
> Do we? Will there be any commands that will be processed by more than one handler?

I think yes, eventually. The goal with the handlers was to make a pattern for handling MDM command results for independent operations. That way we don't end up with 1 mega-function per MDM command result type that tries to contain all possible Fleet logic for that type.